crates/solver/src/settlement_submission/submitter/custom_nodes_api.rs

```diff
@@ -120,11 +123,13 @@ impl TransactionSubmitting for CustomNodesApi {
     }
 
     fn submission_status(&self, settlement: &Settlement, network_id: &str) -> SubmissionLoopStatus {
-        // disable strategy if not mev safe (check done only for mainnet)
-        if !settlement.mev_safe() && shared::gas_price_estimation::is_mainnet(network_id) {
-            return SubmissionLoopStatus::Disabled(DisabledReason::MevExtractable);
+        // disable strategy if there is a slightest possibility for a transaction to be reverted (check done only for mainnet)
+        if shared::gas_price_estimation::is_mainnet(network_id) {
```

Comment:
Why do we only do this for mainnet? In theory this applies to all networks right?

Reply:
Currently on gnosis chain our configuration is to use only `CustomNodes`, so If we were to allow disabling it we would be left without any strategy. Eden and Flashbots rpcs are not enabled on gnosis chain afaik (maybe wrong)

Reply:
Ah! Of course!

Reply:
Although I do agree it seems a bit hardcoded and hard to maintain (it will be easy to forget about this `if` when we add new network).